html/admin/api/projects/crew/crewRoles/applicationList.php

```diff
@@ -0,0 +1,38 @@
+<?php
+require_once __DIR__ . '/../../../apiHeadSecure.php';
+
+if (!$AUTH->instancePermissionCheck(123) or !isset($_POST['projectsVacantRoles_id'])) finish(false, ["code" => "PARAM-ERROR", "message"=> "No data for action"]);
```

Comment:
If they do not have permission but have given data is it not a bit odd to say "no data"

Reply:
like line 12

Reply:
Agreed, have changed